### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by sonradanfransiz**

### CODE REVIEW

**Review by smythi93 — DISMISSED** [review]

File: src/tests4py/projects/markup.py
Comment (smythi93) [inline src/tests4py/projects/markup.py:1]: This is already markup and should be another pull request.

**Comment by sonradanfransiz** [comment]
This has been done already, can be improved in the future.

**Comment by smythi93** [comment]
Why closed?

**Review by smythi93 — APPROVED** [review]
